packages/fast-usdc/src/exos/advancer.js

```diff
@@ -192,15 +202,8 @@ export const prepareAdvancerKit = (
           const { destination, advanceAmount, ...detail } = ctx;
           const transferV = E(poolAccount).transfer(
             destination,
-            {
-              denom: usdc.denom,
-              value: advanceAmount.value,
-            },
-            {
-              forwardOpts: {
-                intermediateRecipient,
-              },
-            },
+            { denom: usdc.denom, value: advanceAmount.value },
```

Comment:
I'm surprised prettier change the second argument too

Reply:
it didn't. I tried a helper for the 1st arg and then took it out, and used the more concise form when I took it out. I realized the second arg didn't need the line-breaks either.